.gitignore

```diff
@@ -0,0 +1,103 @@
+# Source: https://github.com/github/gitignore/blob/master/Android.gitignore
+# Source: https://github.com/github/gitignore/blob/master/Java.gitignore
+# Source: https://github.com/github/gitignore/blob/master/Gradle.gitignore
+
+# Built application files
+*.apk
+*.ap_
+
+# Files for the ART/Dalvik VM
+*.dex
+
+# Java class files
+*.class
+
+# Generated files
+bin/
+gen/
+out/
+
+# Gradle files
+.gradle/
+build/
+
+# Local configuration file (sdk path, etc)
+local.properties
+
+# Proguard folder generated by Eclipse
+proguard/
+
+# Log Files
+*.log
+
+# Android Studio Navigation editor temp files
+.navigation/
+
+# Android Studio captures folder
+captures/
+
+# IntelliJ
+*.iml
+.idea/workspace.xml
```

Comment:
I know this is from Android.gitignore. But why don't they simplify the rules to `.idea/**`? Just curious.